doc/api/perf_hooks.md

```diff
@@ -383,6 +392,18 @@ added: v16.1.0
 An object which is JSON representation of the `performance` object. It
 is similar to [`window.performance.toJSON`][] in browsers.
 
+#### Event: `'resourcetimingbufferfull'`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The `'resourcetimingbufferfull'` event is fired when the global performance
```

Comment:
Could you include the default value somewhere in the docs?

Reply:
Thanks for the suggestion! Updated.